eZ/Publish/Core/Persistence/Cache/CacheServiceDecorator.php

```diff
@@ -24,6 +27,12 @@ class CacheServiceDecorator
      */
     protected $cachePool;
 
+    /** @var int */
+    protected $transactionDepth = 0;
+
+    /** @var array */
+    protected $deferredClear = [];
```

Comment:
Naming: though it's clear here it's an array, when looking over code I felt at some point it was a callback, so I'd make this name more specific:
```suggestion
    protected $keysForDeferredClear = [];
```
or
```suggestion
    protected $deferredClearKeys = [];
```